Resources/config/dbal.xml

```diff
@@ -64,6 +64,7 @@
             <argument>%doctrine.default_entity_manager%</argument>
         </service>
         <service id="Doctrine\Common\Persistence\ManagerRegistry" alias="doctrine" public="false" />
+        <service id="Doctrine\Common\Persistence\ConnectionRegistry" alias="doctrine" public="false" />
```

Comment:
I'm very much against adding more interfaces from `doctrine/persistence` and tying them to ORM. Existing aliases will be dropped in 2.0, see #870. Do we have other options here?

Reply:
Another bundle? :/ What's the reasoning behind not having interfaces from `doctrine/persistence` in this bundle? 🤔 If ORM changes and `doctrine` is not anymore the service holding the `ConnectionRegistry` anymore, we "just" have to release a new version of this bundle isn't it?

Reply:
ORM isn’t the only mapper. The surrounding interfaces will be removed in 2.0 as the assumption that any interface from persistence should be owned by ORM goes out the door once another mapper (e.g. MongoDB ODM) is used.

Reply:
We solved this in general up until now by having an sub-interface that’s ORM-only iirc.

But if that doesn’t exist, it is tricky: not using these interfaces for autowiring optimizes for the 1% that use ORM and ODM simultaneously. 

We could use the named autowiring thing that was added in 4.2 to make it ConnectionRegistry $dbalRegistry (where you need the type + arg name to match)

Reply:
> We could use the named autowiring thing that was added in 4.2 to make it ConnectionRegistry $dbalRegistry (where you need the type + arg name to match)

But it wouldn't be a good DX for the 99% :/